test/deploy.sh

```diff
@@ -59,6 +59,17 @@ function cleanup_and_result() {
    [ -e $RESULT_FILE ] && cat $RESULT_FILE
 }
 
+function debug_staking() {
+   hmy_one_dir="$(go env GOPATH)/src/github.com/harmony-one"
+   hmy_bin="${hmy_one_dir}/go-sdk/hmy"
+   keystore="${hmy_one_dir}/harmony-ops/test-automation/api-tests/LocalnetValidatorKeys"
+   python3 -m pip install pyhmy
```

Comment:
do you have to install it every time? can you check if it is installed?

Reply:
that command only installs if it is not installed